app/src/main/java/com/example/dispatcher/MainActivity.kt

```diff
@@ -17,6 +20,19 @@ class MainActivity : AppCompatActivity() {
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        if (BuildConfig.DEBUG) {
```

Comment:
extract this firebase code to a manager object ("FirebaseCrashlyticsManager"), that will handle all of the related firebase code there